images/rootfs-acrn.yml.in

```diff
@@ -13,6 +13,8 @@ init:
   - GPTTOOLS_TAG
   - DOM0ZTOOLS_TAG
 onboot:
+   - name: mkpersist
```

Comment:
this needs to be combined with how we're handling CONFIG partition (see comments below)

Reply:
Should I rename this container to something like mkconfig-persist?

Reply:
I'd rather avoid mk* prefix since it is confusing with containers that bake binary images (like mkimage-iso-efi and mkimage-raw-efi). I think the right name is an init or something along those lines. So something like storage-init would do.

Reply:
Will change name to storage-init here and other places as applicable.